app/static/js/update_demand_graph.js

```diff
@@ -1,19 +1,60 @@
 /*jshint esversion: 6 */
 // variable to store the timeseries as they are retrieved from the database
 var timeseriesData = {};
+var householdTraces = {};
```

Comment:
I am not sure how this variable gets updated? In the code below it seems to me it is only updated if it is not empty

Reply:
Nevermind I overlooked `householdTiers` for `householdTraces`